op-node/rollup/types.go

```diff
@@ -254,6 +265,16 @@ func (cfg *Config) CheckL2GenesisBlockHash(ctx context.Context, client L2Client)
 	return nil
 }
 
```

Comment:
I would like to call `IsDependencySetUpdate` in the attributes builder and optionally add a new deposit tx